src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -1809,6 +1809,21 @@ public override void Accept(OperationVisitor visitor)
         }
     }
 
+    internal partial class BoundMethodToken
+    {
+        protected override OperationKind ExpressionKind => OperationKind.None;
+
+        public override void Accept(OperationVisitor visitor)
```

Comment:
It's unreachable today. This node, along with others, will need a real implementation when we decide to expose IOperation over lowered trees.